### DISCUSSION TITLE
Set the "audio" format for playlist (option|button)

### DISCUSSION THREAD

**Discussion by Flashwalker** [FreeTubeApp/FreeTube#3230] [General]
How about to add an option or a button to set "audio" format for the playlist (while playing all videos in playlist)?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Initially i wanted to comment with that u can change the video format to audio at the first video in the playlist and when i goes to the next it should still be audio. Unfortunately this isn't the case. I added this usecase to the list in my issue https://github.com/FreeTubeApp/FreeTube/issues/2295

There is no such option at the moment.
